app/src/main/java/de/danoeh/antennapod/fragment/SearchFragment.java

```diff
@@ -51,6 +51,7 @@
  */
 public class SearchFragment extends Fragment {
     private static final String TAG = "SearchFragment";
+
```

Comment:
Please remove the unrelated code change to keep the git history clean (and also reduce the probability of merge conflicts)